RevenueCatUI/CustomerCenter/Views/CompatibilityLabeledContent.swift

```diff
@@ -99,6 +99,17 @@ extension CompatibilityLabeledContent where Label == Text {
     }
 }
 
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+extension CompatibilityLabeledContent where Label == Text, Content == EmptyView {
+    init(_ label: String) {
```

Comment:
Why not just use `Text(label)` in these cases?

Reply:
I find this super convenient, to make the SwiftUI code less messy. 

As an example, this now can be used like:
```swift
CompatibilityLabeledContent("Username") {
    Text("john@example.com")
}
```
instead of:
```swift
CompatibilityLabeledContent(label: { Text("Username") }, content: { Text("john@example.com") })
```

Reply:
Right, but this case of the `init(_ label: String)` where Content is `EmptyView`, you'd be using 

```swift
CompatibilityLabeledContent("Something")
```

whereas, in reality, it's just a `Text("Something")`. So you might as well just use `Text` if you have no content anyway

Reply:
I misunderstood your comment 🤓 

`CompatibilityLabeledContent` takes the whole width, whereas `Text` needs the `.frame(maxWidth: .infinity)`.

Reply:
Oh right. Makes sense. Thank you for the clarification!